README.md

```diff
@@ -1 +1,52 @@
-# the-concert-project
\ No newline at end of file
+# the-concert-project
+
+## Description
+
+- This project is about building a site for an event set up and mobile version first part is creating an entry page to the site of the event
+- The second part is creating speakers exhibition pages and a list of their names titles and description of the program.
+- The third part is the about part which gives an enlightment on the project.
+- The forth part is applying media queries to the site to make the mobile version and desktop version available on different devices and to enhance th UX of the site.
+- The burger menu part aims to create and open a nav menu when clicked on hamburger icon.
+- the pop-up windows part aims to show a pop up window once the user clicks on the burger menu icon at project section.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- node-modules
+- lighthouse
+- hintweb
+- stylint
+
+## links
```

Comment:
- [ ] I suggest adding getting started section, where users will see how to setup your project. You can try this: 


```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/basem909/the-concert-project.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/basem909/the-concert-project.git`.
- go to the repository folder in your command prompt, run `cd the-concert-project`.
- run `npm install`
```

- [ ] Lets keep readme professional and capitalise all headings such as `links, credits to, loom video link `